lib/tests.js

```diff
@@ -44,7 +44,7 @@ export async function runTests({
   // modules, filtered by the pattern specified by the `--grep` CLI option.
   const testFiles = [
     bootstrapFile,
-    ...glob.sync(testsPattern).filter(path => (grep ? path.match(grep) : true)),
+    ...globSync(testsPattern).filter(path => (grep ? path.match(grep) : true)).sort(),
```

Comment:
See https://github.com/hypothesis/frontend-build/pull/266/files#r1132200421

This is specially important here, because I found out tests in `client` do not pass if they are not run in order. To be checked.